ru/contributing/running-the-project.md

```diff
@@ -30,41 +30,40 @@ drupal8.dev \
 --no-interaction
 ```
 
-### Install Drupal using SQLite
+### Установите Drupal, используя SQLite
 ```
 drupal site:install standard --db-type="sqlite" --no-interaction
 ```
-> NOTE: You can install drupal using MySQL by executing the `site:install` command and answering the questions from the interactive mode or passing the required options.
+> ПРИМЕЧАНИЕ: Вы можете установить Drupal, используя MySQL, применив команду `site:install` и ответив на вопросы в интерактивном режиме или передав необходимые параметры.
 
-### Download the Drupal Console Develop package
+### Загрузить пакет Drupal Console Develop
 ```
 composer require drupal/console-develop --dev
 ```
 
-### Create a symbolic link between Drupal and forked repositories
+### Создать символическую ссылку между Drupal и ответвленным репозиторием.
 ```
 drupal develop:create:symlinks \
 --code-directory=/Users/username/drupal-console-code/
 ```
 
-## Downloading additional Drupal Console language or packages
+## Загрузить дополнительные языки или пакеты Drupal Console
 
-If you want to contribute translating Drupal Console to [Spanish](https://github.com/hechoendrupal/drupal-console-es) you should:
+Если вы хотите помочь перевести Drupal Console на [Испанский](https://github.com/hechoendrupal/drupal-console-es) вам следует:
 
-1.- Download it on the drupal site by executing the following command.
+1.- Скачайте его в ваш Drupal сайт с помощью команды.
 
 ```
 composer require drupal/console-es
 ```
 
-2.- Fork and clone the repo to your local directory.
+2.- Создайте fork и склонируйте репозиторий в ваш локальный каталог.
 
-3.- Execute the `develop:create:symlinks` again, to create symlinks including the recently added package.
+3.- Выполните снова команду `develop:create:symlinks`, чтобы создать символические ссылки, включая недавно добавленные пакеты.
 
-This applies for additional languages and packages i.e [drupal/console-yaml](https://github.com/weknowinc/drupal-console-yaml).
+Это применимо к дополнительным языкам и пакеты, например [drupal/console-yaml](https://github.com/weknowinc/drupal-console-yaml).
 
-## Wrapping up
+## Заключение
+Теперь вы можете делать необходимые изменения и начать помогать, делать комиты ваших изменений, отправлять ваш код в ответвленные репозитории и создавать Pull request в соответствующие репозиторий.
 
-Now you can do the required changes and start contributing, commit you changes, push code to your forked repositories and create a Pull Request to the respective repository.
```

Comment:
Please translate this line and the next one.

Reply:
@enzolutions I'm sorry, I have missed an empty line that is why it was looking like I hadn't translated the paragraph, but in fact it was translated. Now everything should be ok.